deploy/utils/convert_api_version.py

```
@@ -0,0 +1,315 @@
+                f"expected {len(indices)}"
+            )
+        for idx, obj in zip(indices, converted):
+            results[idx] = clean_for_authoring(obj)
+
+    return results
+
+
+def _kubectl(args: list) -> str:
+    """Run kubectl and return stdout, raising ConversionError on failure."""
+    try:
+        proc = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)